### DISCUSSION TITLE
Flashing firmware Windows 10 did nothing

### DISCUSSION THREAD

**Discussion by patschgoo** [darthcloud/BlueRetro#816] [General]
### BlueRetro firmware version

1.9

### BlueRetro firmware specification

HW1

### BlueRetro firmware variant

Universal

### BlueRetro hardware type

External adapter with detachable cord

### Manufacturer

BitFunx

### System used

Sony PlayStation 2

### What is problem? (only list ONE problem per report)

I installed the drivers and COM appeared, but after flashing the card with the  3 binary file (1.9 and I also tried 1.5) nothing seems to change. The chip only powers on and the sync butten doesn't blink (except for a short time after I press reset). What should I do now?

![image](https://github.com/darthcloud/BlueRetro/assets/88206373/4bcea0d1-02e1-4c47-8820-13246f552ada)

### What did you expect to happen?

To revert back to it's original state even if it's a older version, but it just doesn't function.

I did everything like explained in:
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10
and
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/establish-serial-connection.html

**Comment by darthcloud** [comment]
If you hold the boot button 10 sec does the led blink fast?

**Reply by patschgoo** [reply]
No it doesn't, the only way i can get the light to turn on is by pressing the EN/Reset button and then it only stays on for as long as I press it or if i press EN/Reset and the Boot button at the same time, then it stays on permanently until I press something else.

**Comment by darthcloud** [comment]
Also.if you got a PlayStation device use the PlayStation firmware. use v1.9 hw1 PlayStation FW

**Reply by patschgoo** [reply]
I did also try that, playstation 1.9 hw1 and universal 1.9 hw1 and the same in v1.5, nothing worked

**Comment by darthcloud** [comment]
[ota_data_initial.bin.zip](https://github.com/darthcloud/BlueRetro/files/13060784/ota_data_initial.bin.zip)

Using the same tool unzip this and write ota_data_initial.bin at 0xd000 (Add a 4th file) and use 1.9 hw1 for the other 3 file (use playstation for 3rd one)

**Reply by patschgoo** [reply]
thank you so much, it's working again ^^
